ci-operator/step-registry/openshift/upgrade/workers-rhel/openshift-upgrade-workers-rhel-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: openshift-upgrade-workers-rhel
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - chain: ipi-aws-pre-workers-rhel
+    - ref: workers-rhel-upgrade
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
+      TEST_TYPE: upgrade
+      TEST_SUITE: ""
+      TEST_UPGRADE_OPTIONS: ""
+  documentation: |-
+    The Openshift E2E AWS workflow executes the upgrade end-to-end test suite on AWS with a default cluster configuration and Loki as log collector, exercising an upgrade during the process.
+    Installer from latest release is used instead. This workflow is intended to verify that changes in installer PR can be used to upgrade the cluster later on.
```

Comment:
It would be good to expand this to describe what the workers-rhel bits are doing.